components/SummaryBox/SummaryBox.tsx

```diff
@@ -222,6 +222,9 @@ const ItemLabel = styled.div`
 
 const ItemText = styled.div`
 	font-size: 1rem;
+	white-space: nowrap;
```

Comment:
This is the fix for this issue: 

<img width="335" alt="Screen Shot 2022-03-17 at 20 35 37" src="https://user-images.githubusercontent.com/15804684/158910811-39135cbf-40f2-4ccb-af91-234977949cb5.png">